src/main/java/com/checkout/instruments/four/create/CreateInstrumentBankAccountResponse.java

```diff
@@ -15,8 +15,6 @@ public final class CreateInstrumentBankAccountResponse extends CreateInstrumentR
 
     private final InstrumentType type = InstrumentType.BANK_ACCOUNT;
 
-    private String fingerprint;
-
     private CustomerResponse customerResponse;
```

Comment:
`customerResponse` was moved to `CreateInstrumentResponse`, so it can be deleted from here right?